SuperBuild/External_CTK.cmake

```diff
@@ -67,13 +67,13 @@ if(NOT DEFINED CTK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/commontk/CTK.git"
+    "${EP_GIT_PROTOCOL}://github.com/pieper/CTK.git"
     QUIET
     )
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "5657c58a72c2db3e9fcc04f88211859eccd21067"
+    "virtualize-database"
```

Comment:
These can be updated once https://github.com/commontk/CTK/pull/1154 has been merged.